go.mod

```diff
@@ -189,30 +189,30 @@ require (
 
 replace (
 	github.com/kcp-dev/kcp/pkg/apis => ./pkg/apis
-	k8s.io/api => github.com/kcp-dev/kubernetes/staging/src/k8s.io/api v0.0.0-20221005071841-6cfb7d485cbf
```

Comment:
I assume you'll drop/replace this commit?

Reply:
Yes, that's the last commit & will be replaced with the fork once we merge the k8s PR